src/jmc/compile/header.py

```diff
@@ -85,6 +88,7 @@ def __clear(obj: "Header"):
         obj.is_enable_macro = True
         obj.namespace_overrides = set()
         obj.commands = set()
+        obj.conditions = VANILLA_CONDITIONS
```

Comment:
This will permanently modify the original dict, you have to copy it.

Reply:
You mean set?

Reply:
Yes, I meant set